tests/models/sho/vmc_sho_hdf5/vmc_sho_hdf5.xml

```diff
@@ -0,0 +1,49 @@
+<simulation>
+  <project id="sho" series="0"/>
+  <random seed="13" />
```

Comment:
Same here. A statistic test should not hard code random seed.

Reply:
I can remove this.